test/suite/test_backup19.py

```diff
@@ -106,5 +106,4 @@ def test_backup19(self):
         self.take_full_backup(self.home_full)
         self.take_incr_backup(self.home_incr)
         self.compare_backups(self.uri, self.home_full, self.home_incr, str(self.bkup_id))
-if __name__ == '__main__':
-    wttest.run()
+
```

Comment:
This one still needs a look - will leave a comment on the other couple of random files that need some attention.